modules/mod_category_tags/src/Helper/CategoryTagsHelper.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  mod_category_tags
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Module\CategoryTags\Site\Helper;
+
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Helper\ContentHelper;
+use Joomla\Database\ParameterType;
+
+/**
+ * Helper for mod_category_tags
+ *
+ * @since  3.1
```

Comment:
this is a new module so the @since cannot be 3.1

we make it easy for you so just use __DEPLOY_VERSION__ and it will be corrected with the exact version on release

```suggestion
 * @since  __DEPLOY_VERSION__
```